internal/broker/bind_create_test.go

```diff
@@ -3,9 +3,120 @@ package broker
 import (
 	"testing"
 
+	"context"
+	"fmt"
+	"net/http/httptest"
+	"time"
+
 	"github.com/stretchr/testify/assert"
+
+	"code.cloudfoundry.org/lager"
+	"github.com/kyma-project/kyma-environment-broker/internal/fixture"
+	"github.com/kyma-project/kyma-environment-broker/internal/storage"
+	"github.com/kyma-project/kyma-environment-broker/internal/storage/dberr"
+	"github.com/pivotal-cf/brokerapi/v8/domain"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+)
+
+type Kubeconfig struct {
+	Users []User `yaml:"users"`
+}
+
+type User struct {
+	Name string `yaml:"name"`
+	User struct {
+		Token string `yaml:"token"`
+	} `yaml:"user"`
+}
+
+const (
+	instanceID1          = "1"
+	instanceID2          = "2"
+	instanceID3          = "max-bindings"
+	expirationSeconds    = 10000
```

Comment:
Why expirationSeconds is greater than maxExpirationSeconds? I bet there is a reason.

Reply:
Params were unnecessary and copied from other test. Corrected by removing them in https://github.com/kyma-project/kyma-environment-broker/pull/1366/commits/805c1ac8e7de479a4d8a554ce2ab2a8b723b44e0.